airflow/bin/cli.py

```diff
@@ -257,6 +258,48 @@ def variables(args):
         print(msg)
 
 
+@cli.action_logging
+def pools(args):
+    """
+    set a batch of Airflow pools
+    :param args: filepath to airflow pools yaml file
```

Comment:
nit: `args` is a parsed args object as produced by ArgParse. It should contain a filepath property.